### PR TITLE
release 3.28.0

### PR DESCRIPTION
**Pull request by RenkuBot**
Release 3.28.0

This PR is auto-generated by [actions/github-script](https://github.com/actions/github-script).

/deploy

### CODE REVIEW

**Review by ciyer — DISMISSED** [review]
🎉

**Review by lorenzo-cavazzi — CHANGES_REQUESTED** [review]
There are a lot of entries [we usually remove from the changelogs](https://github.com/SwissDataScienceCenter/renku-ui/releases), like the libraries version bumps and other chores. We also usually remove stuff like `chore:` or `fix:`

**Comment by leafty** [comment]
> There are a lot of entries [we usually remove from the changelogs](https://github.com/SwissDataScienceCenter/renku-ui/releases), like the libraries version bumps and other chores. We also usually remove stuff like `chore:` or `fix:`

@lorenzo-cavazzi can you make a suggestion or push a commit? This comment is not actionable as is.

**Comment by lorenzo-cavazzi** [comment]
> > There are a lot of entries [we usually remove from the changelogs](https://github.com/SwissDataScienceCenter/renku-ui/releases), like the libraries version bumps and other chores. We also usually remove stuff like `chore:` or `fix:`
> 
> @lorenzo-cavazzi can you make a suggestion or push a commit? This comment is not actionable as is.

@leafty while preparing a suggestion, I realized some of the changes here are features/improvements rather than bugfixes. Here is a PR with both the suggestions for the changelogs and the other changes for making it a full release https://github.com/SwissDataScienceCenter/renku-ui/pull/3183

File: client/package-lock.json
Comment (leafty) [inline client/package-lock.json:1]: needs fixing

**Review by leafty — CHANGES_REQUESTED** [review]

**Comment by leafty** [comment]
Closing in favor of #3184.

**Pull request closed without merging**